rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1053,28 +1053,37 @@ private void migrateFunds(
             Coin amountMigrated = selectedUTXOs.stream()
                 .map(UTXO::getValue)
                 .reduce(Coin.ZERO, Coin::add);
-            pegoutsWaitingForConfirmations.add(btcTx, rskExecutionBlock.getNumber(), rskTxHash);
+            pegoutsWaitingForConfirmations.add(migrationTransaction, rskExecutionBlock.getNumber(), rskTxHash);
             // Log the Release request
             logger.debug(
                 "[migrateFunds] release requested. rskTXHash: {}, btcTxHash: {}, amount: {}",
                 rskTxHash,
-                btcTx.getHash(),
+                migrationTransaction.getHash(),
                 amountMigrated
             );
-            eventLogger.logReleaseBtcRequested(rskTxHash.getBytes(), btcTx, amountMigrated);
+            eventLogger.logReleaseBtcRequested(rskTxHash.getBytes(), migrationTransaction, amountMigrated);
         } else {
-            pegoutsWaitingForConfirmations.add(btcTx, rskExecutionBlock.getNumber());
+            pegoutsWaitingForConfirmations.add(migrationTransaction, rskExecutionBlock.getNumber());
+        }
+
+        if (activations.isActive(RSKIP428)) {
```

Comment:
`if (shouldLogPegoutCreation())`

descriptive names please :))

Reply:
I don't consider this needed here. Even though, if we agree to do this for every RSKIP, I think we should do it in a better way, but not creating this kind of method in all the places where the RSKIP is being checked. Sometimes RSKIP checks are done in more than one class, so it will imply having this kind of method you proposed in more than one place by doing as you proposed.

Reply:
you are right, we should find a better way. I'll resolve this conversation.